src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInPage.java

```diff
@@ -117,6 +135,21 @@ private void isTrackingRequestsNotSend(
       List<String> elementsList, NetworkTrafficInterceptor networkTrafficInterceptor
   ) {
     wait.forX(WAITING_TIME_FOR_ALL_REQUESTS);
+    isConsentCookieSetToRejected();
+    for (String anElementsList : elementsList) {
```

Comment:
`anElementsList` and `elementsList` aren't too descriptive. Could it be `url` and `listOfUrlsToVerify`?